db/python/utils.py

```diff
@@ -431,6 +431,11 @@ def get_logger():
     return _logger
 
 
+def from_db_json(text):
+    """Convert DB's JSON text to Python object"""
+    return json.loads(text)
```

Comment:
I checked the [documentation](https://docs.python.org/3.10/library/json.html#json.load) and it turns out that since py3.6 `json.loads()` can handle `bytes` itself! So I don't think we need that bit inside `from_db_json` (can just depend on `json.loads()` to handle it itself), and can simplify some of the pre-existing code by converting it to use `from_db_json` instead.

Where other calling code would be simplified would be if `from_db_json` also handled None:

```python
if text is None:
    return None
```

We wouldn't need that for external_ids, but it wouldn't hurt either. Any opinion on handling that, @illusional?

Reply:
Fun! I swear we added it because something happened, but if tests pass, I'm happy!

Reply:
Improvements to `from_db_json()` is now a followup PR.